dashboard/config/linux/bits/base.yml

```diff
@@ -0,0 +1,124 @@
+ # Common config fragments required by syzbot for all kernels
+
+verbatim:
+ # This config can be used to enable any additional temporal debugging features in linux-next tree.
```

Comment:
I think this was also copied into the final .config.

If so, it needs to be an actual verbatim block of text using `verbatim: |`

Reply:
I will check what this option does to newlines. I saw most of these multi-line options mess with newlines too much.

Reply:
Done.